salt/soc/defaults.yaml

```diff
@@ -1456,6 +1456,14 @@ soc:
           org: Security Onion
           bucket: telegraf/so_short_term
           verifyCert: false
+        playbook:
+          autoUpdateEnabled: true
+          playbookImportFrequencySeconds: 86400
+          playbookImportErrorSeconds: 600
+          playbookRepoUrl: https://github.com/Security-Onion-Solutions/securityonion-resources
```

Comment:
I'm surprised this works. Usually you need double quotes around values with colons in a .yaml file.